src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.html

```diff
@@ -288,15 +288,27 @@ <h1 class="gct-heading h2">Gene Comparison Tool</h1>
               class="table-divider table-divider-1"
             >
               <div class="table-divider-inner">
-                <div id="pinned-genes-header">Pinned Genes ({{ pinnedGenes.length }}/50)</div>
+                <div id="pinned-genes-header">
+                  <span *ngIf="category === 'RNA - Differential Expression'">
+                    Pinned Genes ({{ pinnedGenes.length }}/50)
+                  </span>
+                  <div *ngIf="category === 'Protein - Differential Expression'">
+                    <div>
+                    Pinned Genes ({{ uniquePinnedGenesCount }}/50)&nbsp;&nbsp;&nbsp;
+                    </div>
+                    <div id="pinned-proteins">
+                      {{ pinnedGenes.length }} Proteins
```

Comment:
Could `pinnedGenes` be changed to `pinnedItems`, since it can contain the list of proteins or genes?

Reply:
@sagely1 also noticed that when `pinnedGenes.length` is 1, the text currently reads `1 Proteins`, but should read `1 Protein`